### ISSUE TITLE
Can't install kink via the tap on a mac m1

### ISSUE THREAD

**Issue by rberrelleza** [Trendyol/kink#20]
I tried installing kink using homebrew, but got the following error:

```
Error: Invalid formula: /opt/homebrew/Library/Taps/trendyol/homebrew-trendyol-tap/kink.rb
formulae require at least a URL
Error: Cannot tap trendyol/trendyol-tap: invalid syntax in tap!
```

This is on a MacM1, MacOS 11.5 and Homebrew 3.2.12

Updated issue to indicate that it's related to arm vs amd64.

**Comment by developer-guy** [comment]
hello @rberrelleza, thanks for opening the issue, but I hate to say that it is working on my machine but really it is, I couldn't reproduce the same issue on my machine:
<img width="1449" alt="Screen Shot 2021-09-17 at 15 27 03" src="https://user-images.githubusercontent.com/16693043/133782375-89643c40-de60-4fac-9b10-e4b717e4998a.png">

PTAL @Dentrax @erkanzileli

**Comment by ysicing** [comment]
@developer-guy  not working in M1

```
==> Tapping trendyol/trendyol-tap
Cloning into '/opt/homebrew/Library/Taps/trendyol/homebrew-trendyol-tap'...
remote: Enumerating objects: 198, done.
remote: Counting objects: 100% (54/54), done.
remote: Compressing objects: 100% (47/47), done.
remote: Total 198 (delta 30), reused 15 (delta 7), pack-reused 144
Receiving objects: 100% (198/198), 37.32 KiB | 796.00 KiB/s, done.
Resolving deltas: 100% (104/104), done.
Error: Invalid formula: /opt/homebrew/Library/Taps/trendyol/homebrew-trendyol-tap/kink.rb
formulae require at least a URL
Error: Cannot tap trendyol/trendyol-tap: invalid syntax in tap!
```

**Comment by rberrelleza** [comment]
You're right, that's the issue. I'll update the issue and the team can decide what to do.

**Comment by ciiiii** [comment]
I'm working on a m1 Mac, can I do some work for this?

**Comment by Dentrax** [comment]
> I'm working on a m1 Mac, can I do some work for this?

As @ysicing said already, we have to build for _arm64_ in the [goreleaser.yaml](https://github.com/Trendyol/kink/blob/3b590455070b28cfa21e1ab7b7f47cab8ad6fc9d/.goreleaser.yml#L16) and update the _brew tap_ afterward.

@ciiiii Yes! You can get to it.

**Comment by ciiiii** [comment]
@Dentrax I do some work about build and installation in PR #47, check that when you guys are free. 😊

**Comment by developer-guy** [comment]
it should be fixed by #47, it will released with v0.2.1
